src/GitExtensions.PluginTemplate/GitExtensions.PluginTemplate.csproj

```diff
@@ -18,12 +19,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GitExtensions.Extensibility" Version="0.1.1.40" />
+    <PackageReference Include="GitExtensions.Extensibility" Version="0.3.0.57" />
+    <PackageReference Include="Microsoft.VisualStudio.Composition" Version="17.4.16" />
+    <PackageReference Include="Microsoft.VisualStudio.threading" Version="17.5.22" />

```

Comment:
Why do we need these two?

Reply:
In Plugin Manager we have `System.ComponentModel.Composition` with `6.0.0` and that's enough. I think we had some problems when referencing VS MEF in Plugin Manager.

Reply:
That's a good point. We can only use versions used in the main app: https://github.com/gitextensions/gitextensions/blob/master/Packages.props#L18-L20
But since we'd be referencing the main app, I'm curious why we need these here.

Reply:
Hm, we just reference main app specific dlls. It doesn't add indirect references to the build, right? And so we need some MEF to be able compile the plugin dll. 

Maybe could add all/some package reference from main app to the GitExtensions.Extensibility

Reply:
I "only" copied those from the `Gerrit` plugin. I was wondering, though, why it was those instead of `System.ComponentModel.Composition`.
I'm perfectly happy to change those for something better. Referencing `Microsoft.VisualStudio.*` feels weird.